benchmarks/viscoelastic_beam_modified/sigma_profile.ipynb

```diff
@@ -0,0 +1,128 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from vtk import vtkUnstructuredGridReader\n",
+    "from vtk import vtkXMLUnstructuredGridReader\n",
+    "from vtk.util import numpy_support as VN\n",
+    "from vtk.util.numpy_support import vtk_to_numpy\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import os"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# A python code that plots the deviatoric stress_xx and stress_yy along a vertical profile near the \n",
+    "# attachment point of the beam. It also plots the analytic solution of sigma_xx from Turcotte and \n",
```

Comment:
Can you add the chapter, equation number, and version of Turcotte and Schubert here?